### PR TITLE
Move to SLE BCI base

### PR DESCRIPTION
**Pull request by macedogm**
Signed-off-by: Guilherme Macedo <guilherme.macedo@suse.com>

### CODE REVIEW

**Review by thedadams — APPROVED** [review]

File: Dockerfile.dapper [hunk 1]
```diff
@@ -1,11 +1,9 @@
-FROM ubuntu:18.04
-# FROM arm=armhf/ubuntu:16.04 arm64=arm64v8/ubuntu:16.04
+FROM registry.suse.com/bci/golang:1.16
```
Comment (thedadams) [inline Dockerfile.dapper:1]: The `go.mod` file in this repo is 1.14. Do we want to bump that too? Or put this at 1.14?

Comment (macedogm) [reply Dockerfile.dapper:1]: We can bump it. I'll extend the PR with this 👍🏻

**Comment by macedogm** [comment]
I'll keep the PR in draft, because we only want to merge after 2.6.5 is out.

File: Dockerfile.dapper [hunk 2]
```diff
@@ -1,12 +1,9 @@
...
-# FROM arm=armhf/ubuntu:16.04 arm64=arm64v8/ubuntu:16.04
+FROM registry.suse.com/bci/golang:1.17
 
 ARG DAPPER_HOST_ARCH
 ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH}
 
-RUN apt-get update && \
-    apt-get install -y gcc ca-certificates git wget curl vim less file && \
-    rm -f /bin/sh && ln -s /bin/bash /bin/sh
+RUN zypper -n install git vim less file curl wget tar gzip
```
Comment (cbron) [inline Dockerfile.dapper:6]: Assuming we don't need to add ca-certificates

**Review by cbron — APPROVED** [review]
Since these changes are more extensive, we need to make sure they get QA'd. Can you writeup the QA template for the various images. That will actually take a good amount of QA time, so the earlier the better.

**Pull request merged**